app/assets/javascripts/admin/services/product_filters.js.coffee

```diff
@@ -0,0 +1,23 @@
+angular.module("ofn.admin").factory "ProductFiltersService", ($httpParamSerializer) ->
```

Comment:
here we should follow local dialect (do whatever the rest of the code is doing) and not name the service service. In this case I think the service could be called ProductFilters or ProductFiltersUrl

Reply:
Done